ethereum/geth_client_test.go

```diff
@@ -46,12 +45,12 @@ type MockTransactionRequest struct {
 func (transactionRequest *MockTransactionRequest) RegisterTransaction(opts *bind.TransactOpts, transactionName string, anotherVar string) (tx *types.Transaction, err error) {
 	transactionRequest.count++
 	if transactionName == "otherError" {
-		err = errors.Wrap("Some other error", 1)
+		err = errors.New("Some other error")
 	} else if transactionName == "optimisticLockingTimeout" {
-		err = errors.Wrap(transactionUnderpriced, 1)
+		err = transactionUnderpriced
```

Comment:
Why not `error.New` here?

Reply:
its already an error constant, no need to do errors.New if you are using error constants, in most cases.